Gems/Atom/Tools/MaterialEditor/Code/Source/Document/MaterialDocument.cpp

```diff
@@ -230,7 +230,6 @@ namespace MaterialEditor
 
         // create source data from properties
         MaterialSourceData sourceData;
-        sourceData.m_propertyLayoutVersion = m_materialTypeSourceData.m_propertyLayout.m_version;
```

Comment:
Required: The MaterialSourceData must have a material type version number to indicate which material type was used when authoring the material. Otherwise it won't know which version updates to apply.